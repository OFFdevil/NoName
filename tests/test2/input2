main
  int A:=123;
    int A:=123;