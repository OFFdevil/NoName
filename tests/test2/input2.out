2
1запускаем отношение в строке с количеством пробелов =  2
